src/Themes/VuetifyTheme/Views/Widget-VList.liquid

```diff
@@ -0,0 +1,5 @@
+<v-list>
```

Comment:
Maybe we should add the v-list props: https://dev.vuetifyjs.com/en/api/v-list/

Other vuetify widgets in the theme can be used as an example of how to handle boolean props. String props can simply be TextFields.